src/FrameHelper.h

```diff
@@ -2,171 +2,225 @@
+        // pixels The size of src must be a multiple of two times dst size otherwise an
+        // exception will be thrown
+        static void resize_bayer(const base::samples::frame::Frame& src,
+            base::samples::frame::Frame& dst,
+            int offset_x = 0,
+            int offset_y = 0);
+
+        // resize a frame
+        // the size is implied by the dst frame
+        static void resize(const base::samples::frame::Frame& src,
+            base::samples::frame::Frame& dst,
+            int offset_x = 0,
+            int offset_y = 0,
+            ResizeAlgorithm algo = INTER_LINEAR);
+
+        // rotates a frame by 180 degrees
+        // dst is inititialized as a copy of src with rotated image
+        static void rotateBy180Degrees(const base::samples::frame::Frame& src,
+            base::samples::frame::Frame& dst);
+
+        // calculates the distance in meters to an object seen on an image
+        // parameters:
+        // f                 focal length of the camera in pixels which has taken the
+        // image
+        //                   set f = fx if the width of the object is used as size
+        //                   set f = fy if the height of the object is used as size
+        // virtual_size      size of the object in pixels
+        // real_size         size of the object in meters
+        static float calcDistanceToObject(float f, float virtual_size, float real_size);
+
+        // calculates the distance in meters to an object seen on an image
+        // parameters:
+        // frame             frame which must have the additional attributes fx (focal
+        // length) and fy virtual_width     width of the object in pixels real_width width
+        // of the object in meters virtual_height    height of the object in pixels
+        // real_height       height of the object in meters
+        // you can set width or height to zero if you do not want to use it
+        // internal always the biggest dimension is used to calculate the distance
+        static float calcDistanceToObject(const base::samples::frame::Frame& frame,
+            float virtual_width,
+            float real_width,
+            float virtual_height,
+            float real_height);
+
+        // calculates the relative positions in meters between to image points
+        // parameters:
+        // fx        focal length of the camera
+        // fy        focal length of the camera
+        // x1        x coordinate of the first point in pixels
+        // y1        y coordinate of the first point in pixels
+        // x2        x coordinate of the second point in pixels
+        // y2        y coordinate of the second point in pixels
+        // d         distance between the camera and the two points
+        static cv::Point2f calcRelPosToPoint(float fx,
+            float fy,
+            int x1,
+            int y1,
+            int x2,
+            int y2,
+            float d);
```

Comment:
I'm not generally against fixing whitespaces (even though it makes reviewing a bit complicated). But maybe this (and similar methods) is an example where one line per parameter is exaggerated ...

Reply:
I use a linter to indent and remove the white spaces.
Just removed the formatting-related commit to facilitate the review